### PR TITLE
Imports

### PR DESCRIPTION
**Pull request by hippogriffic**
This is a messy one, sorry. The first commit is just removing an extra folder that was just adding scaffolding for no reason.

The second commit has more relevant changes.

@incuna/frontend please review

### CODE REVIEW

File: hover/effects/_2d-transitions.sass [hunk 1]
```diff
@@ -0,0 +1,52 @@
+/* 2D TRANSITIONS */
+@import "2d-transitions/grow"
```
Comment (pandalion) [inline hover/effects/_2d-transitions.sass]: do these need to be in quotes?

Comment (hippogriffic) [inline hover/effects/_2d-transitions.sass]: no, this is left over from the original library. I guess I can go through and take them off, might be nicer. I'll do that

**Comment by hippogriffic** [comment]
@pandalion done :)

**Comment by pandalion** [comment]
:+1:  cool !

**Comment by hippogriffic** [comment]
yay thanks :D

**Pull request merged**